### ISSUE TITLE
analyze quadrapad animal with superanimal pretrain model

### ISSUE THREAD

**Issue by juneleung** [DeepLabCut/DeepLabCut#2750]
**Is your feature request related to a problem? Please describe.**, 
related to this issue https://github.com/DeepLabCut/DeepLabCut/issues/2488
I want to analyze a dog video with the pre-trained superanimal model on pytorch framework, when I use `create_pretrained_project` with the `model='superanimal_quadruped'`, it seems some part of pretrained model will not create because the model only support `full_human`. How can I analyze the video of a dog video? Thank you for your cool work!

**Describe the solution you'd like**
I want to connect with **Anipose** to a 3D detection, it seems the `video_inference_superanimal` does not match the file format, I want to use analyze - filter - label ... as the normal pipeline.

**Describe alternatives you've considered**
I am trying to change the result from `video_inference_superanimal` but seems hard to match analyze_videos output format.

**Additional context**
<img width="1556" alt="image" src="https://github.com/user-attachments/assets/9ea72435-7432-43d5-a343-00e1c36401c2">

**Comment by n-poulsen** [comment]
Hi @juneleung! The `create_pretrained_project` method is not yet fully integrated for DeepLabCut 3.0 models. I'll work on integrating the method so that you can follow this workflow, as I think it's a useful addition to DeepLabCut.

**Comment by n-poulsen** [comment]
Closing this issue as it's a duplicate of #2732 - development can be tracked there
